.travis.yml

```diff
@@ -9,6 +9,7 @@ php:
   - 7.1
   - 7.2
   - 7.3
+  - 7.4
   - 7.4snapshot
```

Comment:
Any reason to keep 7.4snapshot now that 7.4 is available?

Reply:
7.4 is not available (i.e. Travis is still failing to build it), but figured we should be monitoring it when it does become so.

Reply:
👍